packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -501,7 +501,7 @@ void main() {
     );
   });
 
-  testWidgets('Top RefreshIndicator showed when dragging from non-zero scroll position', (WidgetTester tester) async {
+  testWidgets('Top RefreshIndicator be shown when dragging from non-zero scroll position', (WidgetTester tester) async {
```

Comment:
neither "showed" nor "be shown" are really correct grammar here, I recommend "shows". :-)

Reply:
(or "is shown")

Reply:
(or "should be shown")

Reply:
Done.
@Hixie Hi, so what's your opinion on this https://github.com/flutter/flutter/issues/9690#issuecomment-298246306 now?

Reply:
copied from discord chat
```
looks like the material design spec doesn't say one way or the other... it seems weird to me to allow pull to refresh when you're just scrolling, since pull to refresh can cost money (it does network traffic) and generally be disruptive. I would not expect it to happen when you're just scrolling, it should be specifically when you're at the top already. I don't have an objection to a flag to enable it when you're just scrolling if that's a common request.
```

I think this is a valid argument, and I am also agree we should have flag to toggle it on and off.